server/models/Profile.js

```diff
@@ -0,0 +1,72 @@
+const mongoose = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'user',
+    unique: true,
+    required: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  gender: {
+    type: String,
+    required: true,
+  },
+  dateOfBirth: {
+    type: Date,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+    trim: true,
+  },
+  phoneNumber: {
+    type: Number,
+    required: true,
+    trim: true,
+  },
+  address: {
+    type: String,
+    required: true,
+    trim: true,
+  },
+  description: {
+    type: String,
+    required: true,
+    trim: true,
+  },
+  type: {
+    type: String,
+    required: true,
+    enum: ['Sitter', 'Owner', 'Sitter/Owner'],
+  },
+  isAvailable: {
+    type: Boolean,
+    required: true,
+  },
+  availability: {
```

Comment:
Are we planning to have a date range (start-end) or availability throughout the month/year (Mon-Fri, each week for eg) ?

Reply:
I think for now we'll deal with a date range. And if any issue arises from that, we'll update accordingly.